draft-ietf-ppm-dap.md

```diff
@@ -1164,14 +1164,11 @@ If obtaining aggregate shares fails, then the leader responds to subsequent HTTP
 GET requests to the collect job URI with an HTTP error status and a problem
 document as described in {{errors}}.
 
-The leader MUST retain a collect job's results until the collector sends an HTTP
-DELETE request to the collect job URI, in which case the leader responds with
-HTTP status 204 No Content.
-
-[OPEN ISSUE: Allow the leader to drop aggregate shares after some reasonable
-amount of time has passed, but it's not clear how to specify that. ACME doesn't
-bother to say anything at all about this when describing how subscribers should
-fetch certificates: https://datatracker.ietf.org/doc/html/rfc8555#section-7.4.2]
+The collector may send an HTTP DELETE request to the collect job URI, to which
+the leader MUST respond with HTTP status 204 No Content. The leader MAY respond
+with HTTP status 204 No Content for requests to a collect job URI which has not
```

Comment:
I think it would be nice to add a little "(e.g. because the data has been deleted due to age or lack of space)". It's a little confusing if you don't have that on your mind.